packages/navlist/src/FzNavlist.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts" setup>
+import { FzRouterNavlinkProps, FzNavlink } from '@fiscozen/navlink'
+import { FzCollapse } from '@fiscozen/collapse'
+import { FzNavlistProps, FzNavlistSub } from './types'
+
+defineProps<FzNavlistProps>()
+
+const isSubMenu = (item: FzRouterNavlinkProps | FzNavlistSub): item is FzNavlistSub => {
+  return Boolean((item as FzNavlistSub).subitems)
+}
+</script>
+
+<template>
+  <div class="fz__navlist inline-flex grow-0 flex-col rounded p-4">
+    <div
+      class="fz__navlist__section border-grey-200"
+      v-for="(section, index) in sections"
+      :key="index"
+    >
+      <div class="text-grey-400 flex h-32 items-center px-12 text-xs">
+        <span>{{ section.label }}</span>
+      </div>
+      <div class="flex flex-col" v-for="(item, itemid) in section.items" :key="itemid">
```

Comment:
itemid is not actually an id, a bit misleading

Reply:
I concur. I also renamed the "index"es in the file to at least have some sort of identifier (there were 3 index for sections, items and subitems)